<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintWidget</class>
 <widget class="QWidget" name="ConstraintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="nameList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="constraintNameLabel">
       <property name="text">
        <string>Constraint name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="constraintNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstJointLabel">
       <property name="text">
        <string>First joint</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="firstJointSelect"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="secondJointLabel">
       <property name="text">
        <string>Second Joint</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="secondJointSelect"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="constraintTypeLabel">
       <property name="text">
        <string>Constraint type</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="constraintTypeSelect"/>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="createConstraintButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="firstGlobalFrame">
       <property name="text">
        <string>Use Global Frame</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="secondGlobalFrame">
       <property name="text">
        <string>Use Global Frame</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>firstGlobalFrame</sender>
   <signal>toggled(bool)</signal>
   <receiver>firstJointSelect</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>secondGlobalFrame</sender>
   <signal>toggled(bool)</signal>
   <receiver>secondJointSelect</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
